app/views/home/landing.html.erb

```diff
@@ -125,9 +125,19 @@
         console.log('onpillclick');
         let $pill = $(this).remove();
         if($pill.data('type') == 'capabilities') {
+          let $oldpill = $("#capabilities").children().first();
+          if ($oldpill) {
+            $('.queryOptionsContainer').filter(':visible').append($oldpill);
```

Comment:
this is basically the same as the one below, could it be refactored into a function?